app/views/createEvent.ejs

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>
+    <%=title %>
+  </title>
+</head>
+<body>
+  <% for(var i in messages) { %>
+    <div class="flash"><%=messages[i] %></div>
+  <% } %>
+  <form action="/createEvent" method="post">
+    <div>
+      <label>Event Type:</label>
+      <input type="text" name="eventType" />
+    </div>
+    <div>
+      <label>Min Number of People:</label>
+      <input type="number" name="numberOfPeopleFrom" />
+    </div>
+    <div>
+      <label>Max Number of People:</label>
+      <input type="number" name="numberOfPeopleTo" />
+    </div>
+    <div>
+      <label>Cost:</label>
+      <input type="number" name="cost" />
+    </div>
+    <div>
+      <label>Deposit:</label>

```

Comment:
Maybe add a lable to describe the meaning of the value. For example: Deposit should be entert as % of the cost...